src/modules/layout/containers/LocationHeader.js

```diff
@@ -57,12 +60,13 @@ export class LocationHeader extends React.Component<PropsType> {
   }
 
   getNavigationItems (): Array<Element<typeof HeaderNavigationItem>> {
-    const { t, isEventsEnabled, isExtrasEnabled, location, events } = this.props
+    const { t, isEventsEnabled, isNewsEnabled, isExtrasEnabled, location, events } = this.props
     const { city, language } = location.payload
     const currentRoute = location.type
 
     const isEventsActive = events ? events.length > 0 : false
     const isCategoriesEnabled = isExtrasEnabled || isEventsEnabled
+    const isNewsActive = true
```

Comment:
Probably this should only be true, if there are news available (just like with events)

Reply:
My idea was to fetch the tunews on the button click and assumed that there will be always data so no need to disable it if there is no push notification.
What do you think?

Reply:
Yeah, I'd agree. There are always TüNews (at least in german), so I guess it's okay to always show it.